docker/docker-compose.yml

```diff
@@ -5,12 +5,16 @@ services:
   damian:
     image: aleph-node:latest
     container_name: damian
+    ports:
+      - 30333:30333
+      - 9933:9933
+      - 9943:9943
```

Comment:
I'm not familiar with docker. Does it change anything?

Reply:
binds these ports from the container to the host network port.

Reply:
Why did you do that now? So that you can reach the locally running or devnet node? Is this just for the devnet, or something more?

Reply:
this compose stack is run only in the CI and locally, devnet is just using the same image, but is based on k8 swarm. CI actually uses another docker container (with curl) to connect to the docker container network and run RPC queries (see bash script for testing finality), because it's easier and faster to pull than installing curl on the CI server. Locally everyone has curl installed so it's nice to expose these ports.

You can even potentially run 4 nodes in a compose  stack and connect a 5-th that is `debug` compiled, record what is going on with a tracer/debugger etc.